src/main/java/org/owasp/esapi/configuration/StandardEsapiPropertyLoader.java

```diff
@@ -70,6 +70,24 @@ public Boolean getBooleanProp(String propertyName) throws ConfigurationException
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Boolean getBooleanProp(String propertyName, Boolean defaultValue) {
+        String property = properties.getProperty(propertyName);
+        if (property == null) {
+            return defaultValue;
+        }
+        if (property.equalsIgnoreCase("true") || property.equalsIgnoreCase("yes")) {
+            return true;
+        }
+        if (property.equalsIgnoreCase("false") || property.equalsIgnoreCase("no")) {
+            return false;
+        }
+        return defaultValue;
```

Comment:
Depending on if we decide to change the behavior here in accordance to my first comment, we may wish to change line 88 to something along the following lines:
```java
    if ( property != null ) {
        throw new ConfigurationException("Property name '" + propertyName + "' has non-Boolean value of '" + property + "'.");
    }
```